### DISCUSSION TITLE
Premake Batch File Munged, Documentation Incorrect

### DISCUSSION THREAD

**Discussion by ChaosOrdeal** [bulletphysics/bullet3#3538] [General]
Just downloaded ver 2.85.1 for Win.  Following the Quickstart pdf, as recommended in main documentation.  Paragraph 2.1.1 in pdf refers to non-existent directories and files.  Guessing at the actual meaning led me to run vs2010_bullet2gpu.bat, which generated error  about "--bullet2gpu" being an invalid switch.  Removed this switch from the batch file, and Premake4 ran and generated files -- too early to say if they're the right files, etc.

The documentation and the batch file need attention.  Realize that for anyone trying Bullet for the first time (or the first time in awhile) on Windows, this issue will be their first impression.

**Comment by erwincoumans** [comment]
On Windows, there are some batch files in the root folder of Bullet. Good point, we should update the documentation and remove the broken/out-of-date vs2010_bullet2gpu.bat, file.

Thanks for the report!
